index.html

```diff
@@ -29,12 +35,133 @@ <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
             <div class="social">
                 <p>let's connect</p>
                 <ul class="social-icon">
-                    <li><img src="./image/twister.png" alt="twister" class="contact-icon"></li>
-                    <li><img src="./image/phone.png" alt="phone" class="contact-icon"></li>
-                    <li><img src="./image/mail.png" alt="mail" class="contact-icon"></li>
-                    <li><img src="./image/facebook.png" alt="facebook" class="contact-icon"></li>
-                </ul>
+                    <li><a href="https://twitter.com/HIENPHAN0111"><i class="fa-brands fa-twitter contact-icon fa-xl"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/hien-phan-61097b256/"><i class="fa-brands fa-linkedin-in contact-icon fa-xl"></i></a></li>
+                    <li><a href="https://github.com/hienphan0111"><i class="fa-brands fa-github contact-icon fa-xl"></i></a></li>
+                    <li><a href="http://"><i class="fa-brands fa-angellist contact-icon fa-xl"></i></a></li>   
+                </ul>                
+            </div>
+        </section>
+        <section class="work">
+            <div class="project pr-1">
+                <div class="snap-shoot">
+                    <img src="./image/Snapshoot Portfolio-1.svg" alt="project 1">
+                </div>
+                <div class="info">
+                    <h2 class="pj-title pt-1">Tonic</h2>
+                    <ul class="meta">
+                        <li class="type">Canopy</li>
+                        <li class="catagories gray"><span class="dot"></span>Backend Dev</li>
+                        <li class="year gray"><span class="dot"></span>2015</li>
+                    </ul>
+                    <p class="project-des">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <ul class="langue">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button class="read-more"><a href="http://"> See Project</a></button>
+                </div>
+            </div>
```

Comment:
- [ ] I think adding the button effects is better:mag: . The button (e.g., "See project") is a link or button tag, and follow the button styling: Active, Hover, and Pressed; see design below.

> ![Screenshot from 2022-12-02 15-24-34](https://user-images.githubusercontent.com/65129228/205292550-81cad857-25ab-49f2-90cb-1ba07124f205.png)